examples/openthread/README.md

```diff
@@ -0,0 +1,69 @@
+## OpenThread on RIOT
+
+This example demonstrates how to use the [OpenThread](https://github.com/openthread/openthread)
+open source implementation of [Thread](https://threadgroup.org/) on RIOT.
+The [Command Line Interface](https://github.com/openthread/openthread/blob/master/examples/apps/cli/README.md) of
+OpenThread was ported. Please check the [full documentation]
+(https://github.com/openthread/openthread/blob/master/src/cli/README.md)
```

Comment:
The link to full documentation is broken, see [here](https://github.com/jia200x/RIOT/tree/openthread_ftd/examples/openthread#openthread-on-riot)
If you want to separate the paragraph, a new line is not enough, you have to add a blank line.

Reply:
right. Just pushed the fix (https://github.com/RIOT-OS/RIOT/pull/9336/commits/fd96a55f9ab4a9d5c220910931b94fc3e41f97fc)